ts/nni_manager/training_service/kubernetes/kubeflow/kubeflowTrainingService.ts

```diff
@@ -310,7 +312,7 @@ class KubeflowTrainingService extends KubernetesTrainingService implements Kuber
         // Generate kubeflow job resource config object
         const kubeflowJobConfig: any = await this.generateKubeflowJobConfig(trialJobId, trialWorkingFolder, kubeflowJobName, workerPodResources,
                                                                       nonWorkerResources);
-
+        this.log.info(`${kubeflowJobConfig}`)
```

Comment:
No string formatting...
Just `this.log.info('kubeflowJobConfig:', kubeflowJobConfig);`